Client/Frontend/Intro/IntroViewController.swift

```diff
@@ -308,93 +247,122 @@ class IntroViewController: UIViewController, UIScrollViewDelegate {
+        titleLabel.adjustsFontSizeToFitWidth = true
+        titleLabel.minimumScaleFactor = IntroUX.MinimumFontScale
+        titleLabel.textAlignment = .center
+        return titleLabel
+    }()
+
+    lazy var textLabel: UILabel = {
+        let textLabel = UILabel()
+        textLabel.numberOfLines = 5
+        textLabel.adjustsFontSizeToFitWidth = true
+        textLabel.minimumScaleFactor = IntroUX.MinimumFontScale
+        textLabel.textAlignment = .center
+        textLabel.lineBreakMode = .byTruncatingTail
+        return textLabel
+    }()
+
+    lazy var button: UIButton = {
+        let button = UIButton()
+        button.backgroundColor = IntroUX.SignInButtonColor
+        button.setTitle(Strings.SignInButtonTitle, for: [])
+        button.setTitleColor(.white, for: [])
+        button.clipsToBounds = true
+        return button
+    }()
+
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        axis = .vertical
+        distribution = .equalSpacing
+        alignment = .center
+        spacing = 10
+        addArrangedSubview(titleLabel)
+        addArrangedSubview(textLabel)
+        alpha = 0
     }
 
-    fileprivate func attributedStringForLabel(_ text: String) -> NSMutableAttributedString {
-        let paragraphStyle = NSMutableParagraphStyle()
-        paragraphStyle.lineSpacing = IntroViewControllerUX.CardTextLineHeight
-        paragraphStyle.alignment = .center
-
-        let string = NSMutableAttributedString(string: text)
-        string.addAttribute(NSParagraphStyleAttributeName, value: paragraphStyle, range: NSRange(location: 0, length: string.length))
-        return string
+    func configureWith(card: IntroCard) {
+        titleLabel.text = card.title
+        textLabel.text = card.text
+        if let buttonText = card.buttonText, card.buttonSelector != nil {
+            addArrangedSubview(button)
+            button.setTitle(buttonText, for: .normal)
+        } else {
+            // We need a blank view to make sure the page controls dont get hidden by autolayout
+            let blankView = UIView()
+            blankView.snp.makeConstraints { make in
+                make.size.equalTo(IntroUX.PageControlHeight)
+            }
+            addArrangedSubview(blankView)
+        }
     }
-    
-    fileprivate func setupDynamicFonts() {
-        startBrowsingButton.titleLabel?.font = UIFont(name: "FiraSans-Regular", size: DynamicFontHelper.defaultHelper.IntroStandardFontSize)
-        signInButton.titleLabel?.font = UIFont(name: "FiraSans-Regular", size: DynamicFontHelper.defaultHelper.IntroBigFontSize)
 
-        for titleLabel in titleLabels {
-            titleLabel.font = UIFont(name: "FiraSans-Medium", size: DynamicFontHelper.defaultHelper.IntroBigFontSize)
+    // Allows the scrollView to scroll while the CardView is in front
+    override func point(inside point: CGPoint, with event: UIEvent?) -> Bool {
+        if let buttonSV = button.superview {
```

Comment:
Nit: minor risk? perhaps the button could get nested in a future commit and the person not realize point(inside:with) will fail

Reply:
We were doing something similar before and at first its pretty odd why it exists. But when you miss around with it you realize very quickly why its there. The intro slides become impossible to scroll right away.